### PR TITLE
Add if statement to skip tapcell insertion for HVL cells (OpenFASOC)

### PR DESCRIPTION
**Pull request by shuolai**
I need to use the high voltage library for OpenFASOC's LDO. And since the sky130hvl technology already contains tapcells in its standard cells, I added an if statement to skip inserting tapcells.

### CODE REVIEW

**Comment by eder-matheus** [comment]
@shuolai your update looks fine, but the CI is failing in tools that are unrelated to your changes. Could you try to update your branch with the latest master branch?

File: src/tap/src/tapcell.tcl [hunk 1]
```diff
@@ -191,11 +191,13 @@ proc tapcell { args } {
     }
   }
 
+
+  if { [info exists keys(-tapcell_master)] } {
   tap::clear
```
Comment (arlpetergadfort) [inline src/tap/src/tapcell.tcl:196]: I appears to me that adding this just bypasses the `tapcell` command all together. It seems to me that the proper solution is to not call `tapcell`, if you don't want tapcells inserted.

Comment (eder-matheus) [reply src/tap/src/tapcell.tcl:196]: @arlpetergadfort You're right. @shuolai  To skip the `tapcell` insertion using the flow, you can create a custom `tapcell.tcl` file for your design that will not call the `tapcell` command.

Comment (shuolai) [reply src/tap/src/tapcell.tcl:196]: @arlpetergadfort @eder-matheus Thank you for your advice. My issue is that I would like to `insertEndcaps` and skip `insertTapcells` at the same time. So I think I have to call `tapcell` function. I tried to remove `tapcell_master` in `tapcell.tcl` but I got the following error message.

Maybe adding a if statement to skip line 156 and 157 in tapcell.cpp could be a solution to this problem? 

tapcell_tcl
error

**Review by eder-matheus — APPROVED** [review]

**Comment by rovinski** [comment]
Wouldn't it make sense to convert all of the masters to `odb::master*` rather than just `tapcell_master`?

**Pull request merged**